config/environments/test.rb

```diff
@@ -37,6 +37,17 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  config.action_mailer.default_url_options = { host: 'localost:3000' }
+
+  config.action_mailer.smtp_settings =  {
+      address: "smtp.example.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.